pkgs/servers/web-apps/vikunja/api.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitea }:
+
+buildGoModule rec {
+  pname = "vikunja-api";
+  version = "0.17.0";
+
+  src = fetchFromGitea {
+    domain = "kolaente.dev";
+    owner = "vikunja";
+    repo = "api";
+    rev = "v${version}";
+    sha256 = "sha256-A9ZrJlGT2QZ9qPz/9XPgsBJqv1PA9Kb5scnXWzZnosc=";
+  };
+
+  vendorSha256 = "sha256-/vXyZznGxj5hxwqi4sttBBkEoS25DJqwoBtADCRO9Qc=";
+
+  # checks need to be disabled because of needed internet for some checks
+  doCheck = false;
+
```

Comment:
According to [build instructions](https://vikunja.io/docs/build-from-sources/), it should be built with Mage ([example](https://github.com/jtojnar/nixfiles/blob/28a348c486e749f15be23f59e7cbea8efb08378f/pkgs/vikunja/vikunja-api/default.nix)). I am surprised this even builds – in the past, I had to [build templates](https://kolaente.dev/vikunja/api/issues/134).

Reply:
We got rid of the template build step since a while ago.

Not building with mage won't set the version and may fail with sqlite as that needs cgo to work correctly (I guess the test would catch that though) so I think while building with a plain `go build .` (I'm assuming `buildGoModule` does only that?) will probably work, it would be better to switch the build to mage.